packages/universal-components/src/Button/__tests__/index.test.js

```diff
@@ -24,7 +24,7 @@ const type = 'info';
 const testID = 'testID';
 const circled = false;
 const size = 'normal';
-
+const style = {};
```

Comment:
why was this change necessary? it seems like it's polluting the snapshots?

Reply:
I assumed that all the props should be in the snapshot, should I remove it in separate PR?

Reply:
It's that repeating question about what we are testing and how. One of the tests is for example `should have passed props` so it probably makes sense there (if we really want to test it).
And same for snapshots. Currently we are using `snapshotDiff` between Button with all props and without. Same as before, question is if it is really useful for something.

Reply:
Yes, we would need to sit down and decide what we believe is the best practice.